misk/src/main/kotlin/misk/digester/MergingDigest.kt

```diff
@@ -0,0 +1,351 @@
+package misk.digester
+
+import misk.proto.tdigest.Centroid
+import misk.proto.tdigest.MergingDigestData
+import kotlin.math.PI
+import kotlin.math.asin
+
+class MergingDigest {
+
+    //amount of compression to data
+    var compression: Double = 0.0
```

Comment:
Make this and other member variables private since they are internal to the class's behavior. `private var compression: Double = 0.0`

By default in Kotlin, omitting a visibility modifier implies `public`.

Reply:
imo these should be `val`s and specified in the ctor